data_dir/large_collections.yaml

```diff
@@ -19,6 +19,9 @@ table_definition: |
     nums_set set<int>,
     names_set set<text>,
     books_set set<text>,
+    static_data1 blob static,
+    static_data2 blob static,
+    PRIMARY KEY(pk_id, user_name)
```

Comment:
why do you also use user_name as PK?

Reply:
using a static column requires using a clustering key.

Reply:
OK.